mmv1/third_party/terraform/tests/resource_kms_crypto_key_test.go

```diff
@@ -440,6 +440,91 @@ func testAccCheckGoogleKmsCryptoKeyRotationDisabled(t *testing.T, projectId, loc
 	}
 }
 
+func TestAccKmsCryptoKeyVersion_basic(t *testing.T) {
```

Comment:
There also needs to be a test for version deletion separate from crypto key deletion, since deleting the crypto key implicitly deletes the versions.

I think the easiest way to do this would be to add a step here where the configuration is like `testGoogleKmsCryptoKeyVersion_basic` but the version has been removed. At that point - I don't think you'd be able to do an ImportStateVerify but you might be able to do an explicit check of the API object, sort of similar to [`testAccCheckGoogleProjectHasBillingAccount`](https://github.com/hashicorp/terraform-provider-google/blob/3069f7581c887e9783e0026a210378d883fc3131/google/resource_google_project_test.go#L304). Note that this is an unusual situation and I'm not entirely sure what the best practice for checking this would be.

Reply:
Sure I will add a second step specifically for deletion.